### ISSUE TITLE
img Tag Refactor: _includes/about-page/about-card-sustainability.html - sdg-la.svg

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#5171]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We want to change an img HTML tag ending with a slash (<img.../>) to an img tag without an ending slash (<img...>) so that the codebase is consistent with how we use img HTML tags.

### Action Items
- [ ] In your local IDE, navigate to `_includes/about-page/about-card-sustainability.html`
- [ ] Remove the ending slash in the img HTML tag by changing
```
<img src="/assets/images/sdg/sdg-la.svg" alt="" />
```
to
```
<img src="/assets/images/sdg/sdg-la.svg" alt="">
```
- [ ] The file may contain more than one instance of an img tag with a closing slash.  Only change the single instance specified above.  Do not leave any blank spaces before the closing bracket.
- [ ] Using Docker, check the page remains the same in mobile, tablet, and desktop views as on the current website (See 1 in the Resources/Instructions section below)

### Resources/Instructions
1. Webpage(s): https://www.hackforla.org/about
2. This issue is tracked in [HfLA-Website: img Tag Audit spreadsheet's img with Ending Slash tab](https://docs.google.com/spreadsheets/d/1cq8iLOYQkAbho2GsRAxVJYgjxDeJ9KLdx2NuL4G6vu4/edit#gid=0), which is part of #4362

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe I am not sure how the inclusion of this 

> For QA to validate change: https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects.js

helps me QA the change

**Comment by roslynwythe** [comment]
@ExperimentsInHonesty that was part of the template before I was assigned to the epic so I'm not sure I know exactly the reasoning behind it but I believe the idea is that you could browse to that file, search for the image filename specified in the issue title, and view the updated code.  Please advise re: a better means of supporting QA.

**Comment by aidanwsimmons** [comment]
i. Availability: Available Monday-Wednesday before 6pm
ii. ETA: Should be completed today

**Comment by aidanwsimmons** [comment]
Fixes #5171

### What changes did you make?
  - Removed '/' from end of img tag

### Why did you make the changes (we will use this info to test)?
  - refactored so that the codebase is consistent with how we use img HTML tags

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

No visual changes to the website.

**Comment by aidanwsimmons** [comment]
removed self- didnt see my previous issue required changes
